src/browser.js

```diff
@@ -168,6 +168,11 @@ function load() {
 
 function localstorage() {
   try {
+    // TVMLKit (Apple TV JS Runtime) does not have a window object, just localStorage in the global context
+    if (typeof window === 'undefined' && typeof navigationDocument !== 'undefined') {
+      return localStorage;
+    }
+
     return window.localStorage;
```

Comment:
Maybe we could just change this line to `localStorage` and have no `if` check?

Reply:
That would definitely work for the TVMLKit scenario, does that work for the general browser scenario as well?

Reply:
`window` is the global object in the web browser, so I can't really think of a scenario where it would be a problem. Only one way to find out though, haha.

Reply:
Yeah I concur. That would work fine.

Reply:
Ok, I'll make that change